src/tools/tracejob.c

```diff
@@ -481,13 +488,35 @@ parse_log(FILE *fp, char *job, int ind)
 
 			if (tmp.date != NULL) {
 				log_lines[ll_cur_amm].date = strdup(tmp.date);
-				if (sscanf(tmp.date, "%d/%d/%d %d:%d:%d", &tms.tm_mon, &tms.tm_mday, &tms.tm_year, &tms.tm_hour, &tms.tm_min, &tms.tm_sec) != 6)
-					log_lines[ll_cur_amm].date_time = -1;	/* error in date field */
-				else {
-					if (tms.tm_year > 1900)
-						tms.tm_year -= 1900;
-					tms.tm_mon--;         /* The number of months since January, in the range 0 to 11 for mktime */
-					log_lines[ll_cur_amm].date_time = mktime(&tms);
+				if ((ind != IND_ACCT) && (strchr(tmp.date, '.'))) {
+					if (sscanf(tmp.date, "%d/%d/%d %d:%d:%d.%ld", &tms.tm_mon, 
+					    &tms.tm_mday, &tms.tm_year, &tms.tm_hour, &tms.tm_min, 
+					    &tms.tm_sec, &(log_lines[ll_cur_amm].highres_ts)) != 7) {
+						log_lines[ll_cur_amm].date_time = -1;	/* error in date field */
+						log_lines[ll_cur_amm].highres_ts = -1;
+					} else {
+						has_high_res_timestamp = 1;
```

Comment:
We have macros for True and False right? can you please use them instead of 1 and 0 literals?

Reply:
It appears we don't have global macros for true and false.  There are some localized macros in different header files, but none of them are appropriate for tracejob to include.